apps/src/sites/studio/pages/teacher_dashboard/index.js

```diff
@@ -16,7 +15,7 @@ $(document).ready(function () {
       <Router basename={baseUrl}>
         <Route
           path="/"
-          component={props => <TeacherDashboard {...props} sectionId={sectionId} studioUrlPrefix={studioUrlPrefix}/>}
+          component={props => <TeacherDashboard {...props} sectionId={sectionId} studioUrlPrefix=""/>}
```

Comment:
you shouldn't need studioUrlPrefix in SectionProjectsListWithData or TeacherDashboard, since those components are only ever rendered on dashboard. But i think this is fine for now, and you can just remove all mentions of studioUrlPrefix once we're no longer using the pegasus teacher dashboard.

Reply:
Thanks! Cleaning up mentions of studioUrlPrefix is on my to do list for the switch from pegasus to dashboard.